src/App.jsx

```diff
@@ -9,32 +10,27 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const defaultTab = tabs[0].id;
+  const [selectedTabId, setSelectedTabId] = useState(defaultTab);
+  const tabTitle = tabs.find(tab => tab.id === selectedTabId).title;
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  const onTabSelected = (tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (tab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364